doc/srv6/srv6_vpn.md

```diff
@@ -0,0 +1,284 @@
+# SRv6 VPN HLD
```

Comment:
The HLD is focusing on more than VPN. Will be good to reflect that in the title to include Policy and BFD as well. Or may be split HLD into 3 different ones. Either way is fine.

Reply:
Policy and BFD are required for VPN deployment. The reason for introducing VPN is for service based traffic steering.